.github/CODEOWNERS

```diff
@@ -1 +1,7 @@
-/.github/workflows/ @JuliaLang/github-actions
+/.github/ @JuliaLang/github-actions 
+/.buildkite/ @JuliaLang/github-actions
+/src/ @JuliaLang/compiler
```

Comment:
Perhaps this should be a bit more fine-grained? I'd be comfortable reviewing frontend changes, but likely won't be able to contribute much if it touches codegen or parts of the C API. Also should perhaps add `base/compiler`?

Reply:
What do you think of my suggestion in https://github.com/JuliaLang/julia/pull/40925#issuecomment-846560766

Reply:
Yeah, that maybe. Although especially for some files in `base` this might be a bit of a noisy measure and it might be better for folks to be able to opt in manually.

Reply:
Won't people's global preferences on notifications anyways override this?

Reply:
I meant noisy here mostly in terms of that it might be rather arbitrary who is assigned to a file or not, but perhaps that's something we should just try and then see how it goes.